jsonschema/schema_test.go

```diff
@@ -54,9 +54,9 @@ func TestJSONRoundTrip(t *testing.T) {
 	for _, tt := range []struct {
 		in, want string
 	}{
-		{`true`, `{}`}, // boolean schemas become object schemas
-		{`false`, `{"not":{}}`},
-		{`{"type":"", "enum":null}`, `{}`}, // empty fields are omitted
+		{`true`, `true`},
```

Comment:
Do we want to make both of these changes, or just 'false'?
I've not seen many examples of a schema being 'true', but have seen examples of a schema being 'false'.